src/main/java/com/microexam/classapi/entities/Class.java

```diff
@@ -0,0 +1,30 @@
+package com.microexam.classapi.entities;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hibernate.annotations.CreationTimestamp;
+import org.springframework.data.annotation.LastModifiedDate;
+
+import javax.persistence.*;
+import java.sql.Timestamp;
+
+@Entity
+@Table(name = "class")
+@Getter @Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class Class {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "class_id")
+    private int classId;
+    @Column(name = "class_name")
```

Comment:
I think its fine to define the column names but by default Entity Manager roots camel casing to underscore casing.

Reply:
I prefer to define the column names